### PR TITLE
ColladaLoader: Added support for TGA-textures

### LINKED ISSUE #14003: Collada Loader TGA support
The example collada loader doesn't appear to support TGA files, I made a fork and with the following changes the loader now supports TGA. Is this something that could be merged?

https://github.com/mrdoob/three.js/compare/dev...vlakken:dev

##### Three.js version

- [ ] Dev
- [x] r92
- [ ] ...

##### Browser

- [x] All of them
- [ ] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [x] All of them
- [ ] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

### PR DESCRIPTION
**Pull request by Mugen87**
Fixes #14003

### CODE REVIEW

File: examples/js/loaders/ColladaLoader.js [hunk 1]
```diff
@@ -969,11 +969,11 @@ THREE.ColladaLoader.prototype = {
 
 			if ( data.bindShapeMatrix ) {
 
-				build.bindMatrix = new THREE.Matrix4().fromArray( data.bindShapeMatrix ).transpose(); 
+				build.bindMatrix = new THREE.Matrix4().fromArray( data.bindShapeMatrix ).transpose();
 
 			} else {
 
-				build.bindMatrix = new THREE.Matrix4().transpose(); 
+				build.bindMatrix = new THREE.Matrix4().transpose();
```
Comment (WestLangley) [inline examples/js/loaders/ColladaLoader.js]: Granted this is just cleanup, but for clarity, I think it should be
```javascript
build.bindMatrix = new THREE.Matrix4().identity();
```
even though the matrix is initialized to the identity by default.

**Comment by Mugen87** [comment]
I hope it's okay to merge this one since I'd like to make a follow up PR.

**Comment by Mugen87** [comment]
@vlakken Thanks for the initial code 👍

File: examples/js/loaders/ColladaLoader.js [hunk 2]
```diff
@@ -969,11 +969,11 @@ THREE.ColladaLoader.prototype = {
 
 			if ( data.bindShapeMatrix ) {
 
-				build.bindMatrix = new THREE.Matrix4().fromArray( data.bindShapeMatrix ).transpose(); 
+				build.bindMatrix = new THREE.Matrix4().fromArray( data.bindShapeMatrix ).transpose();
 
 			} else {
 
-				build.bindMatrix = new THREE.Matrix4().transpose(); 
+				build.bindMatrix = new THREE.Matrix4().identity();
```
Comment (makc) [inline examples/js/loaders/ColladaLoader.js]: neither transpose nor identity make sense on new matrix 😕

Comment (Mugen87) [reply examples/js/loaders/ColladaLoader.js]: Please read https://github.com/mrdoob/three.js/pull/14475#discussion_r202517423

It's their for clarity.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**